src/lib/Menu/MainMenuBuilder.php

```diff
@@ -175,27 +176,28 @@ public function createStructure(array $options): ItemInterface
         /** @var \Knp\Menu\ItemInterface|\Knp\Menu\ItemInterface[] $menu */
         $menu = $this->factory->createItem('root');
 
-        $menu->addChild($this->factory->createItem(self::ITEM_DASHBOARD, [
-                'route' => 'ezplatform.dashboard',
+        $menu->addChild($this->factory->createItem(self::ITEM_CONTENT, [
```

Comment:
+1 for suggested change

for side note: those are different factories `\Knp\Menu\MenuFactory` and `\EzSystems\EzPlatformAdminUi\Menu\MenuItemFactory`

Reply:
@damianz5 makes sense.

The issue of using wrong factory when using `addChild()` without an already constructed `Item` can be circumvented by slightly changing `EzSystems\EzPlatformAdminUi\Menu\MenuItemFactory::createItem`:
```diff
- return $this->factory->createItem($name, array_merge_recursive($defaults, $options));
+ $item = $this->factory->createItem($name, array_merge_recursive($defaults, $options));
+ $item->setFactory($this);
+
+ return $item;
```

This would ensure that when calling `$item->addChild()` our implementation of the MenuFactory is called, not the default one.